### PR TITLE
[WIP][POAE7-287]Add shell to detect AEP so as to cache backend fallback

### PR DESCRIPTION
**Pull request by Aaxida**

### CODE REVIEW

**Comment by Aaxida** [comment]
@offthewall123  Could you please help to review this prototype?

File: dev/detect_pmem.sh [hunk 1]
```diff
@@ -0,0 +1,3 @@
+sudo -S sudo ipmctl show -dimm << EOF 
+password
+EOF
```
Comment (Aaxida) [inline dev/detect_pmem.sh:3]: Need to rewrite to the password of sudo.

Comment (offthewall123) [reply dev/detect_pmem.sh:3]: Any better solution? It's not elegant for user to offer every nodes's password in the cluster

Comment (offthewall123) [reply dev/detect_pmem.sh:3]: Add license header.

File: oap-cache/oap/src/main/scala/org/apache/spark/sql/execution/datasources/oap/filecache/MemoryManager.scala [hunk 2]
```diff
@@ -127,6 +127,12 @@ private[sql] object MemoryManager extends Logging {
 
 
   def apply(sparkEnv: SparkEnv, memoryManagerOpt: String): MemoryManager = {
+    var detectPmemShPath = System.getProperty("user.dir") + "/dev/detect_pmem.sh"
```
Comment (offthewall123) [inline oap-cache/oap/src/main/scala/org/apache/spark/sql/execution/datasources/oap/filecache/MemoryManager.scala:130]: Need to import scala.language.postfixOps?

File: oap-cache/oap/src/main/scala/org/apache/spark/sql/execution/datasources/oap/filecache/OapCache.scala [hunk 3]
```diff
@@ -218,6 +218,12 @@ private[filecache] object OapCache extends Logging {
                           fallBackRes: Boolean = true): Boolean = {
     if (fallBackEnabled == false && fallBackRes == true) return true
     if (fallBackEnabled == false && fallBackRes == false) return false
+    var detectPmemShPath = System.getProperty("user.dir") + "/dev/detect_pmem.sh"
```
Comment (offthewall123) [inline oap-cache/oap/src/main/scala/org/apache/spark/sql/execution/datasources/oap/filecache/OapCache.scala:221]: Same as above.

**Comment by offthewall123** [comment]
Add [OAP-CACHE] to the title and submit an issue and add issue id. Can refer to this PR's title 
https://github.com/Intel-bigdata/OAP/pull/1740

File: dev/detect_pmem.sh (hunk 1 above)
Comment (Aaxida) [reply dev/detect_pmem.sh:3]: Got it! I'll manage to find a better way.

**Pull request closed without merging**